src/test/features/claim/routes/check-and-send.ts

```diff
@@ -81,7 +87,221 @@ describe('Claim issue: check and send page', () => {
+      })
+
+      it('Should validate that a claim made by individual against organisation and displays their details on check-and-send page', async () => {
+        draftStoreServiceMock.resolveFind('claim',
+          { defendant: { ...draftStoreServiceMock.sampleClaimDraftObj.defendant, partyDetails: organisationDetails } })
+        feesServiceMock.resolveCalculateIssueFee()
+
+        await request(app)
+          .get(ClaimPaths.checkAndSendPage.uri)
+          .set('Cookie', `${cookieName}=ABC`)
+          .expect(res => expect(res).to.be.successful.withText('<a href="/claim/defendant-organisation-details" class="bold">Change <span class="visuallyhidden">full name</span></a>'))
+          .expect(res => expect(res).to.be.successful.withText('Organisation.'))
+      })
+
+      it('Should validate that a claim made by soleTrader against soleTrader and displays their details on check-and-send page', async () => {
+        draftStoreServiceMock.resolveFind('claim',
+          { claimant: { ...draftStoreServiceMock.sampleClaimDraftObj.claimant, partyDetails: soleTraderDetails }, defendant: { ...draftStoreServiceMock.sampleClaimDraftObj.defendant, partyDetails: soleTraderDetails } })
+        feesServiceMock.resolveCalculateIssueFee()
+
+        await request(app)
+          .get(ClaimPaths.checkAndSendPage.uri)
+          .set('Cookie', `${cookieName}=ABC`)
+          .expect(res => expect(res).to.be.successful.withText('<a href="/claim/claimant-sole-trader-details" class="bold">Change <span class="visuallyhidden">full name</span></a>'))
+          .expect(res => expect(res).to.be.successful.withText('<a href="/claim/defendant-sole-trader-details" class="bold">Change <span class="visuallyhidden">full name</span></a>'))
+          .expect(res => expect(res).to.be.successful.withText('Business name'))
+          .expect(res => expect(res).to.be.successful.withText('Trading as SoleTrader Ltd.'))
+      })
+
+      it('Should validate that a claim made by soleTrader against individual and displays their details on check-and-send page', async () => {
+        draftStoreServiceMock.resolveFind('claim',
+          { claimant: { ...draftStoreServiceMock.sampleClaimDraftObj.claimant, partyDetails: soleTraderDetails } })
+        feesServiceMock.resolveCalculateIssueFee()
+
+        await request(app)
+          .get(ClaimPaths.checkAndSendPage.uri)
+          .set('Cookie', `${cookieName}=ABC`)
+          .expect(res => expect(res).to.be.successful.withText('<a href="/claim/claimant-sole-trader-details" class="bold">Change <span class="visuallyhidden">full name</span></a>'))
+          .expect(res => expect(res).to.be.successful.withText('Business name'))
+          .expect(res => expect(res).to.be.successful.withText('Trading as SoleTrader Ltd.'))
+          .expect(res => expect(res).to.be.successful.withText('<a href="/claim/defendant-individual-details" class="bold">Change <span class="visuallyhidden">full name</span></a>'))
```

Comment:
To be consistent with the other tests should this not also be checking for the name of the individual?

Reply:
adding checking for the name of the individual